plugin/src/main/kotlin/com/strumenta/kolasu/languageserver/plugin/LanguageServerPlugin.kt

```diff
@@ -219,7 +227,7 @@ class LanguageServerPlugin : Plugin<Project?> {
                         [
                             {"id": "${configuration.language}", "extensions": ["${configuration.fileExtensions.joinToString(
                     "\", \""
-                ){ ".$it" }}"]$fileIcon}
+                ){ ".$it" }}"]$fileIcon$languageConfig}
```

Comment:
Consider collecting elements in a list (without commas) and then join them inserting a string.

Something like:
```listOf( {"id": "${configuration.language}", "extensions": ["${configuration.fileExtensions.joinToString(
                    "\", \""
                ){ ".$it" }}"], fileIcon, languageConfig).join(", )```